### ISSUE TITLE
Fig Integration: Figterm is not running in this terminal, please try restarting your terminal

### ISSUE THREAD

**Issue by hochgenug** [withfig/fig#2059]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

After reinstalling Fig, expect that `fig doctor` command run well 

### Actual behaviour

`fig doctor`output : 
```
✘ Fig Integration: Figterm is not running in this terminal, please try restarting your terminal
  FIG_TERM=
✘ Doctor found errors. Please fix them and try again.
```
I use iterm2, tried to restart my Mac, terminal, fig. And reinstall many time fig to be sure.

Output of the verbose version of the doctor command : 
✔ Logged into Fig

Let's check your dotfiles...
✔ ~/.bashrc contains valid fig hooks
✔ ~/.profile contains valid fig hooks
✔ ~/.zshrc contains valid fig hooks
✔ ~/.zprofile contains valid fig hooks

Let's make sure Fig is setup correctly...
✔ Fig bin exists
✔ PATH contains ~/.local/bin
✔ PATH contains ~/.fig/bin
✔ Settings Corruption
✔ State Corruption
✘ Fig Integration: Figterm is not running in this terminal, please try restarting your terminal
  FIG_TERM=

Let's make sure Fig is running...
✔ Fig is running
✔ Fig socket exists
✘ Daemon: Daemon is not running
✘ Daemon diagnostics: Daemon socket exists but could not connect
  Socket path: /var/tmp/fig/jho/daemon.socket
  Io(Os { code: 61, kind: ConnectionRefused, message: "Connection refused" })

Let's see if Fig is in a working state...
✘ Figterm: No FIGTERM_SESSION_ID

   0: No FIGTERM_SESSION_ID
   1: environment variable not found

Location:
   fig_cli/src/cli/doctor.rs:655

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
✔ Insertion lock does not exist
✔ Autocomplete dev mode
✔ Plugin dev mode
✔ Dashboard is loading from the correct URL
✔ Autocomplete is loading from the correct URL

Let's check if your system is compatible...
✔ OS is supported
✔ Fish is up to date
✔ Jetbrains Toolbox Check

Let's check fig diagnostic...
✔ Compatible shell
✔ Fig app installed in the right place
✔ Autocomplete is enabled
✔ Fig CLI path
✔ Accessibility enabled

Output of `~/.zshrc`

```
# Fig pre block. Keep at the top of this file.
[[ -f "$HOME/.fig/shell/zshrc.pre.zsh" ]] && builtin source "$HOME/.fig/shell/zshrc.pre.zsh"
# If you come from bash you might have to change your $PATH.
# export PATH=$HOME/bin:/usr/local/bin:$PATH

# Path to your oh-my-zsh installation.
export ZSH="$HOME/.oh-my-zsh"
export DEFAULT_USER="$(whoami)"

ZSH_THEME="agnoster"
plugins=(git)

source $ZSH/oh-my-zsh.sh
# Fig post block. Keep at the bottom of this file.
[[ -f "$HOME/.fig/shell/zshrc.post.zsh" ]] && builtin source "$HOME/.fig/shell/zshrc.post.zsh"
```

### Environment

```yaml
fig-details:
  - 2.7.9
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/jho/www
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - SHELL: /bin/zsh
    - PATH: /Users/jho/.nvm/versions/node/v16.17.0/bin:/Users/jho/Library/Python/3.10/bin:/opt/homebrew/opt/openjdk@11/bin:/opt/homebrew/opt/make/libexec/gnubin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/.local/bin:/.fig/bin:/Users/jho/.pulumi/bin
    - TERM: xterm-256color
```

**Comment by grant0417** [comment]
Does Fig work if you run `figterm` in you terminal directly? Not 100% sure why it isn't launching by default, can you also share your `~/.zprofile`

**Comment by hochgenug** [comment]
I got a `command not found: figterm` for figterm.

Here my  `~/.zprofile` file
```
# Fig pre block. Keep at the top of this file.
[[ -f "$HOME/.fig/shell/zprofile.pre.zsh" ]] && builtin source "$HOME/.fig/shell/zprofile.pre.zsh"
eval "$(/opt/homebrew/bin/brew shellenv)"

# Fig post block. Keep at the bottom of this file.
[[ -f "$HOME/.fig/shell/zprofile.post.zsh" ]] && builtin source "$HOME/.fig/shell/zprofile.post.zsh"
```

**Comment by PleahMaCaka** [comment]
I have the same issue but after running 'figterm' it works fine. Where should `figterm` be run? zshrc stuff?

**Comment by hochgenug** [comment]
For information I tried a new installation again with the .dmg from the website and also with brew but still stuck with the figterm issue.

**Comment by mikemajara** [comment]
@hochgenug did you close coz you found a solution? Your last comment suggests you are still facing a problem. It'd be useful to understand what happened.

I'm facing the same problem (although not the first hiccup): @grant0417 in my case fig does work if I execute figterm. But not with the default configuration, I also had to manually add `$HOME/.local/bin`, and `$HOME/.fig/bin` to my path in `.zshrc`

**Comment by hochgenug** [comment]
Hello @mikemajara,
Yeah I found what was the issue ! It was because I had two installations of Fig on my Mac (one from the .dmg downloaded on the website and one with brew called "Fig 2" in my applications)

First, I removed the all the stuff related to Fig.
Then check the symbolic links for fig and figterm : `ls -als ~/.fig/bin` (in my case, the links were on the Fig 2 application that was removed)
Also do the same check of the bin : `ls -als ~/.local/bin`

It was working well after fixing that manually. A new installation didn't overwrite these links.
